src/transformStateWithClones.js

```diff
@@ -66,6 +66,31 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const newState = { ...state };
+  const transformedStates = [];
+
+  for (const { operation, properties } of transforms) {
+    switch (operation) {
+      case 'addProperties':
+        Object.assign(newState, properties);
+        break;
+      case 'removeProperties':
+        for (const key of properties) {
+          delete newState[key];
+        }
+        break;
+      case 'clear':
+        for (const key in newState) {
+          delete newState[key];
+        }
+        break;
+      default:
```

Comment:
usually in switch case construction developers adds default case, it's a common issue that there is no default case, but it would be nice to have something like 
```
default: 
  // Do nothing
```

or 
```
default: 
 Error.throw('there is no such a case')
```